libraries/wavemap/include/wavemap/integrator/integrator_factory.h

```diff
@@ -1,22 +1,26 @@
 #ifndef WAVEMAP_INTEGRATOR_INTEGRATOR_FACTORY_H_
 #define WAVEMAP_INTEGRATOR_INTEGRATOR_FACTORY_H_
 
+#include <memory>
 #include <string>
 
 #include "wavemap/data_structure/volumetric/volumetric_data_structure_base.h"
 #include "wavemap/integrator/integrator_base.h"
+#include "wavemap/utils/thread_pool.h"
 
 namespace wavemap {
 class IntegratorFactory {
  public:
   static IntegratorBase::Ptr create(
       const param::Value& params,
       VolumetricDataStructureBase::Ptr occupancy_map,
+      std::shared_ptr<ThreadPool> thread_pool = nullptr,
       std::optional<IntegratorType> default_integrator_type = std::nullopt);
 
   static IntegratorBase::Ptr create(
       IntegratorType integrator_type, const param::Value& params,
```

Comment:
Maybe put the second parameter onto a line of it's own.

Reply:
Do you mean `std::shared_ptr<ThreadPool> thread_pool = nullptr` (already a single param) or maybe another line?

Reply:
The delta view showed some other parameter being tacked to the back of the thread_pool line for some reason, though here it's not longer showing it that way. So who knows what the github thing did then.